stable/external-dns/README.md

```diff
@@ -68,6 +68,7 @@ The following table lists the configurable parameters of the external-dns chart
 | `publishInternalServices`          | Allow external-dns to publish DNS records for ClusterIP services (optional).                                               | `false`                                            |
 | `rbac.create`                      | If true, create & use RBAC resources                                                                                       | `false`                                            |
 | `rbac.serviceAccountName`          | Existing ServiceAccount to use (ignored if rbac.create=true)                                                               | `default`                                          |
+| `interval`                         | Interval update period to use (options: txt, noop)                                                                                | `txt`                                              |
```

Comment:
This looks like a copy/paste error. As per `values.yaml`, the default is `1m`, and the value should be a time duration, so `options: txt, noop` is wrong too.